capstone/src/main/java/com/google/univiz/SuggestionApiReaderProvider.java

```diff
@@ -0,0 +1,8 @@
+package com.google.univiz;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+interface SuggestionApiReaderProvider {
+  InputStream getStreamFromUrl(String urlString) throws IOException;
```

Comment:
This interface and it's implementation are doing the same thing as `CollegeIdReaderProvider` and `CollegeIdReaderProviderImpl` in #82. Unnecessary duplication is probably bad, so we should stick with either your interface or mine.